Actions/AnalyzeTests/TestResultAnalyzer.ps1

```diff
@@ -23,27 +23,32 @@ function BuildTestMarkdownTable {
         [hashtable] $resultIcons
     )
 
+    $mdTableRows = [System.Collections.ArrayList]@()
     $mdTableSB = [System.Text.StringBuilder]::new()
 
     foreach($row in $Rows) {
         Write-Host $row
+        $formattedRow = @()
         for($i=0; $i -lt $row.Length; $i++) {
             #If resultIcons has a key for this column, we want to display an emoji for values > 0 and an empty cell for 0
             if ($resultIcons.ContainsKey($i)) {
                 if ($row[$i] -gt 0) {
-                    $row[$i] = "$($row[$i])$($resultIcons[$i])"
+                    $formattedRow += "$($row[$i])$($resultIcons[$i])"
                 }
                 else {
-                    $row[$i] = $null
+                    $formattedRow += $null
                 }
+            } else {
+                $formattedRow += $row[$i]
             }
         }
-        Write-Host $row
+        $mdTableRows.Add($formattedRow) | Out-Null
+        Write-Host $formattedRow
```

Comment:
Maybe `"|$($formattedRow -join '|')|" | Out-Host` instead?
to balance the number of lines in the log?
(And the above Write-Host $row - could be `$row -join ',' | Out-Host`